app/containers/ChangeLocale/index.js

```diff
@@ -49,41 +49,43 @@ export const ChangeLocale = ({ locale, changeLocaleDispatch, withTitle }) => {
   if (process.env.MULTI_LANG === 'false') return null;
 
   return (
-    <Dropdown
-      className="mr-3"
-      button={
-        <React.Fragment>
-          <Span
-            className="d-flex align-items-center mr-1"
-            fontSize="16"
-            lineHeight="20"
-            color={TEXT_SECONDARY}
-          >
-            <Flag src={require(`images/${[locale]}_lang.png`)} alt="country" />
-            {withTitle && <FormattedMessage {...commonMessages[locale]} />}
-          </Span>
-        </React.Fragment>
-      }
-      menu={
-        <ul>
-          {appLocales.map(x => (
-            <Li
-              key={x}
-              role="presentation"
-              onClick={() => setLocale(x)}
-              isBold={x === locale}
-            >
-              <Flag src={require(`images/${x}_lang.png`)} alt="language" />
-              <FormattedMessage {...commonMessages[x]} />
-            </Li>
-          ))}
-        </ul>
-      }
-      id="choose-language-dropdown"
-      isArrowed
-      isMenuLabelMobile
-      isArrowMarginMobile
-    />
+    <>
+    </>
+    // <Dropdown
```

Comment:
could you please add TODO with ticket number, because it's not obvious why this code is commented

Reply:
I made a mistake in the logic of the task.
I had to remove the language switcher but not the whole component.
On the call they said to do it temporarily.
Now everything is fixed.

Reply:
good, checked it, looks fine

Reply:
thanks